AISKU/snippet/snippet.js

```diff
@@ -320,9 +321,9 @@
     // name: "appInsights", // Global SDK Instance name defaults to "appInsights" when not supplied
     // ld: 0, // Defines the load delay (in ms) before attempting to load the sdk. -1 = block page load and add to head. (default) = 0ms load after timeout,
     // useXhr: 1, // Use XHR instead of fetch to report failures (if available),
-    // crossOrigin: "anonymous", // When supplied this will add the provided value as the cross origin attribute on the script tag
+    crossOrigin: "anonymous", // When supplied this will add the provided value as the cross origin attribute on the script tag
     // onInit: null, // Once the application insights instance has loaded and initialized this callback function will be called with 1 argument -- the sdk instance (DO NOT ADD anything to the sdk.queue -- As they won't get called)
     cfg: { // Application Insights Configuration
-        instrumentationKey: "INSTRUMENTATION_KEY"
+        connectionString: "InstrumentationKey=INSTRUMENTATION_KEY"
```

Comment:
Yeh, this is because this snippet is consumed by the web-snippet package and we have not updated that yet, so this keeps it compatible with that (rather than having 2 versions). This will change in the coming months.

Reply:
Actually it consumes the `snippet.min.js`...